src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/AutomationTests.ps1

```diff
@@ -361,3 +361,62 @@ function Test-AutomationStartUnpublishedRunbook
     Remove-AzureRmAutomationRunbook $accountName -Name $runbook.Name -Force 
     Assert-Throws {Get-AzureRmAutomationRunbook $accountName -Name $runbook.Name -Parameters $runbookParameters -PassThru -ErrorAction Stop}
 }
+
+<#
+.SYNOPSIS
+Tests create new automation variable with Get-Content
+#>
+function Test-CreateNewVariableWithGetContent
+{
+	$resourceGroupName = Get-RandomResourceGroupName
+	$rgLocation = "japaneast"
```

Comment:
do not hard code the location - use a get loctaion call to get an appropriate location - hard coding the location causes issues with running tests live, and issues with the next dev re-recording the tests.  See: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-developer-guide.md#using-common-code  for common code fixes for this

Reply:
I will fix this.